docker-compose.yaml

```diff
@@ -1,5 +1,7 @@
 version: '3.7'
 
+name: myapp
```

Comment:
Let's name it somehow instead 'myapp'

Reply:
It is a leftover and should be deleted (in the previous version, we filtered logs from containers that weren't included in the docker compose project, but we don't have a use case for that, so I decided to remove it)